rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1521,6 +1514,50 @@ public Coin getMinimumLockTxValue() {
         return bridgeConstants.getMinimumLockTxValue();
     }
 
+    /**
+     * Votes for a fee per kb value.
+     *
+     * @return 2 upon successful vote with election winner,
+     * 1 upon successful vote without winner,
+     * -1 when the vote failed,
+     * FEE_PER_KB_GENERIC_ERROR_CODE when there was an un expected error.
+     */
+    public Integer voteFeePerKbChange(Transaction tx, Coin feePerKb) {
+        TxSender voter = TxSender.fromTx(tx);
+        ABICallElection feePerKbElection = provider.getFeePerKbElection(bridgeConstants.getFeePerKbChangeAuthorizer());
+        ABICallSpec feeVote = new ABICallSpec("setFeePerKb", new byte[][]{BridgeSerializationUtils.serializeCoin(feePerKb)});
```

Comment:
Probably not the best to use `BridgeSerializationUtils` functions at this level, but can refactor in the future.

Reply:
Also, we're double checking authorization in the federation change voting so that we return a different value (the equivalent to `FEE_PER_KB_GENERIC_ERROR_CODE`) when not authorized. Can we do this here too?